cli/running/running.go

```diff
@@ -371,3 +364,24 @@ func Check(cmdCtx *cmdcontext.CmdCtx) error {
 
 	return nil
 }
+
+// CreateInstance reads config and creates instance.
+func CreateInstance(cmdCtx *cmdcontext.CmdCtx, args []string) (*Instance, error) {
```

Comment:
Suppose, this function must not be exported from running package.

Reply:
Changed